### ISSUE TITLE
ComplexWarning

### ISSUE THREAD

**Issue by peterstangl** [DavidMStraub/ckmutil#6]
The currently published version (v1.1.0) does not contain the merged PR #5 and therefore produces many `ComplexWarning: Casting complex values to real discards the imaginary part`. @DavidMStraub could you please release a new version so we get rid of all these warnings? Thanks :)

**Comment by DavidMStraub** [comment]
While we are at it, should we move it to the `flav-io`` namespace as well?

**Comment by peterstangl** [comment]
Yes, I think that would make sense! But maybe we could do this separately from this issues since moving to `flavi-io` would involve several steps (moving the repo, moving the api doc, updating the PyPI project, etc.) and maybe would take a while?

(I actually have an issue with PyPI at the moment since I don't have access to the primary email address of my account anymore and even though I have a verified secondary email set up, the primary one is not verified and so I don't have access to my account. I started account recovery one month ago but this seems to take several months. I now created a new account https://pypi.org/user/pstangl/)
